src/main/java/Main.java

```diff
@@ -0,0 +1,17 @@
+// overloading in methods
+class A {
```

Comment:
i tolk to move this into separete class or file.
why you write 2 classes in same file.